README.md

```diff
@@ -104,7 +104,7 @@ In the GitHub repository the full documentation is available, including:
 
 You can read the documentation for the kdb tool, either
 
-- [on the Website](https://www.libelektra.org/manpages/kdb)
+- [on the Website](https://www.libelektra.org/man pages/kdb)
```

Comment:
I think the path is specified in `src/tools/website/resources/structure.json.in`

```suggestion
- [on the Website](https://www.libelektra.org/man-pages/kdb)
```

Reply:
Nice catch, thank you very much